.github/crates-filters.yml

```diff
@@ -1,23 +1,3 @@
-consensus-config:
-  - "consensus/config/**"
-consensus-core:
-  - "consensus/core/**"
-iota-common:
-  - "crates/iota-common/**"
-iota-metrics:
-  - "crates/iota-metrics/**"
-iota-network-stack:
-  - "crates/iota-network-stack/**"
-iota-util-mem:
```

Comment:
`util-mem` and `util-mem-derive` also exist in the repo.

Reply:
They were just moved

Reply:
yeah I'm blind, sorry 😅